vcf-dumper/vcf-dumper-ws/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>embl.ebi.variation.eva</groupId>
+        <artifactId>vcf-dumper</artifactId>
+        <version>0.1</version>
+    </parent>
+
+    <artifactId>vcf-dumper-ws</artifactId>
+    <packaging>war</packaging>
+
+    <dependencies>
```

Comment:
removed the connected ones. There are two spring boot dependencies, still have to review if they both are needed